.github/workflows/pull-request.yaml

```diff
@@ -34,6 +34,11 @@ jobs:
           - os: linux
             run: ubuntu-18.04
             targets: ci-go-ci-build-linux ci-go-ci-build-linux-static
+            arch: amd64
+          - os: linux
+            run: ubuntu-18.04
+            targets: ci-go-ci-build-linux ci-go-ci-build-linux-static
+            arch: arm64
```

Comment:
Let's add the `matrix.arch` to the build name (line 27), it looks hard to discern right now:

![image](https://user-images.githubusercontent.com/870638/150757482-458945ae-9987-4944-82e3-7bed87cb191a.png)

Reply:
We'll probably have to update the "required builds" then, but that's OK.

Reply:
Updated. How's this?

<img width="221" alt="Screen Shot 2022-01-24 at 10 45 02 AM" src="https://user-images.githubusercontent.com/1031317/150826469-302ff0de-736f-46f7-abf8-994722f18fee.png">

Reply:
Thanks